src/main/java/org/benetech/servicenet/matching/AlternateNameSimilarityCounter.java

```diff
@@ -0,0 +1,18 @@
+package org.benetech.servicenet.matching;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AlternateNameSimilarityCounter extends AbstractSimilarityCounter<String> {
+
+    @Value("${similarity-ratio.weight.alternate-name}")
```

Comment:
I think it would be good to make the ratio independent from the similarity matcher - applying ratios should be the responsibility of the aggregator.